mani_skill/agents/robots/trifingerpro/trifingerpro.py

```diff
@@ -0,0 +1,176 @@
+from copy import deepcopy
+from typing import List
+
+import sapien
+import torch
+
+from mani_skill import PACKAGE_ASSET_DIR
+from mani_skill.agents.base_agent import BaseAgent
+from mani_skill.agents.controllers import *
+from mani_skill.agents.registration import register_agent
+from mani_skill.agents.utils import (
+    get_active_joint_indices,
+)
+from mani_skill.utils.sapien_utils import (
+    get_objs_by_names,
+)
+from mani_skill.utils.structs.pose import vectorize_pose
+
+
+@register_agent()
+class TriFingerPro(BaseAgent):
+    """
+    Modified from https://github.com/NVIDIA-Omniverse/IsaacGymEnvs/blob/main/isaacgymenvs/tasks/trifinger.py
+
+    """
+    uid = "trifingerpro"
+    urdf_path = f"{PACKAGE_ASSET_DIR}/robots/trifinger/trifingerpro.urdf"
```

Comment:
it doesn't seem like these assets are in the repo? can you add them

Reply:
I think I did add it. Isn't it the right place to put it? https://github.com/haosulab/ManiSkill2/blob/7c6d1d64ada6a4c06ce9a23c0a468f85350a74dd/mani_skill/assets/robots/trifinger/trifingerpro.urdf

Reply:
must have missed the file.